docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md

```diff
@@ -2,6 +2,7 @@
 
 | Version | Date |
 |---|---|
+| [v2.1.0](Data_Portal_Release_Notes.md#release-210) | April 30, 2024 |
```

Comment:
we'll hv to change this if launch date changes.
I assumed we would call it 2.1.0, but up to Bill, really.

Reply:
2.1 is good